deployment/ansible/files/cosmovisor_preupgrade.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
thanks for moving that inside `ansible` folder.
but it would be great to make that a part of the bootstrap role (`bootstrap/files`)